### ISSUE TITLE
quitting gradio, team does not care about breaking changes every week! themes are down too!

### ISSUE THREAD

**Issue by cowanAI** [gradio-app/gradio#6332]
### Describe the bug

Just venting out, insane how what was supposed to be progress with gradio 4.0 turned into pure embarrasement with our users, we are used to the breaking changes,  but cmon, if you are going to take down the themes which is crucial for the identity of the apps, at least give a heads up. 

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

Just venting out, insane how what was supposed to be progress with gradio 4.0 turned into pure embarrasement with our users, we are used to the breaking changes,  but cmon, if you are going to take down the themes which is crucial for the identity of the apps, at least give a heads up. 

### Screenshot

Just venting out, insane how what was supposed to be progress with gradio 4.0 turned into pure embarrasement with our users, we are used to the breaking changes,  but cmon, if you are going to take down the themes which is crucial for the identity of the apps, at least give a heads up. 

### Logs

```shell
Just venting out, insane how what was supposed to be progress with gradio 4.0 turned into pure embarrasement with our users, we are used to the breaking changes,  but cmon, if you are going to take down the themes which is crucial for the identity of the apps, at least give a heads up.
```

### System Info

```shell
Just venting out, insane how what was supposed to be progress with gradio 4.0 turned into pure embarrasement with our users, we are used to the breaking changes,  but cmon, if you are going to take down the themes which is crucial for the identity of the apps, at least give a heads up.
```

### Severity

Blocking usage of gradio

**Comment by abidlabs** [comment]
Hi @cowanAI I'm sorry to hear about the issues you're facing! It is true that Gradio 4.0 has had breaking changes compared to 3.x. We've made a list of breaking changes here as well as advised on how to migrate: https://github.com/gradio-app/gradio/blob/main/CHANGELOG.md#breaking-changes 

These breaking changes were necessary because we refactored a significant portion of the Gradio codebase for 4.0 in order to let users customize Gradio apps even further, and also allowed us to clean up unused and deprecated portions of the code for better maintainability going forward. 

In terms of the specific issue that you mentioned, which is that themes stopped working, this is actually not due to breaking changes in Gradio 4.0, but rather due to a bug in one of our dependencies. We issued a fix as soon as we became aware of the problem (https://github.com/gradio-app/gradio/issues/6298), and the issue **should now be fixed in Gradio 4.1.2**. To reiterate, we are **not** deprecating themes, we recognize the importance of being able to customize your Gradio apps, and we've taken steps to increase that flexibility in Gradio 4.0, not decrease it. 

Thanks @cowanAI for being a Gradio user and contributing actively to the issues and discussion.
